### ISSUE TITLE
Upgrade broke bind

### ISSUE THREAD

**Issue by cygnus8595** [skycocker/chromebrew#2475]
## Description
Ran `crew upgrade` and ran the following upgrades:
```
libunbound could be updated from 1.7.0 to 1.7.3
icu4c could be updated from 60.1 to 62.1
```

Now, whenever I run any bind utilities - like dig or nslookup - I get the following error:

```
chronos@localhost ~/Downloads $ dig
dig: error while loading shared libraries: libicui18n.so.60: cannot open shared object file: No such file or directory
chronos@localhost ~/Downloads $ nslookup
nslookup: error while loading shared libraries: libicui18n.so.60: cannot open shared object file: No such file or directory
```

## Versions and system information
chromebrew: 0.4.3

ChromeOS: `cat /etc/lsb-release`
CHROMEOS_AUSERVER=https://tools.google.com/service/update2
CHROMEOS_BOARD_APPID={BABC2E82-8F7D-26E5-EE66-4C60A2B41DBC}
CHROMEOS_CANARY_APPID={90F229CE-83E2-4FAF-8479-E368A34938B1}
CHROMEOS_DEVSERVER=
CHROMEOS_RELEASE_APPID={BABC2E82-8F7D-26E5-EE66-4C60A2B41DBC}
CHROMEOS_RELEASE_BOARD=candy-signed-mp-v2keys
CHROMEOS_RELEASE_BRANCH_NUMBER=58
CHROMEOS_RELEASE_BUILDER_PATH=candy-release/R67-10575.58.0
CHROMEOS_RELEASE_BUILD_NUMBER=10575
CHROMEOS_RELEASE_BUILD_TYPE=Official Build
CHROMEOS_RELEASE_CHROME_MILESTONE=67
CHROMEOS_RELEASE_DESCRIPTION=10575.58.0 (Official Build) stable-channel candy 
CHROMEOS_RELEASE_NAME=Chrome OS
CHROMEOS_RELEASE_PATCH_NUMBER=0
CHROMEOS_RELEASE_TRACK=stable-channel
CHROMEOS_RELEASE_VERSION=10575.58.0
DEVICETYPE=CHROMEBOOK
GOOGLE_RELEASE=10575.58.0

Model: Dell Chromebook 11 (candy)

Architecture: x86_64

## Addtional information
I'm guessing the bind package needs to be upgraded to match the library versions upgraded above

**Comment by JL2210** [comment]
This ties into icu4c, too. See #2470

**Comment by skiman6010** [comment]
Please re-open the issue if #2483 doesn't fix this problem.

**Comment by michaelcadilhac** [comment]
As far as I can see, the problem is back:

```
$ crew upgrade
Your software is already up to date.
$ crew whatprovides /usr/local/lib64/libicui18n.so
icu4c: /usr/local/lib64/libicui18n.so
icu4c: /usr/local/lib64/libicui18n.so.67
icu4c: /usr/local/lib64/libicui18n.so.67.1

Total found: 3
$ ldd /usr/local/bin/host | grep libicui
        libicui18n.so.62 => not found
        libicui18n.so.67 => /usr/local/lib64/libicui18n.so.67 (0x0000787b84e2d000)
$ host
host: error while loading shared libraries: libicui18n.so.62: cannot open shared object file: No such file or directory
```
I don't know why `host` is linked with two versions of libicui18n though.  The same holds for libicuuc.so.62 and libicudata.so.62.  Symlinking libicu*.so.62 to libicu*.so solves the problem.

**Comment by uberhacker** [comment]
Fixed in #4181.
